911.js

```diff
@@ -152,11 +152,11 @@ function getTrafficData(src) {
     $.getJSON(src, function (data) {
         data.forEach(dp => {
             let description = "<p>"
-                + dp.description
-                + "</br></br> This driver is from: "
-                + dp.driver_city + " "
+                + dp.description[0] + dp.description.slice(1).toLowerCase()
```

Comment:
I liked the function better – this is a little messy, but you can leave that for a follow-up.

Reply:
#remindMe in 4 years to update